playwright/tests/pageobjects/CodeEditor.ts

```diff
@@ -36,6 +38,16 @@ class CodeEditor {
     await expect(this.code).toBeVisible();
   }
 
+  async openBrowsers() {
+    await this.focus();
+    await this.scriptArea.getByLabel('Browser').click();

```

Comment:
Fine for me. But I think you could also find the button via getByRole:
`this.scriptArea..getByRole('button', { name: 'Browser' });`